projects/1.19/assets/heat-and-climate/dcs_climate/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+  "block.dcs_climate.build.slab_granite":"黑花岗岩台阶",
+
+  "block.dcs_climate.build.wall_mud":"泥岩墙",
+  "block.dcs_climate.build.wall_gypsum":"石膏墙",
+  "block.dcs_climate.build.wall_serpentine":"蛇纹岩墙",
+  "block.dcs_climate.build.wall_greisen":"云英岩墙",
+  "block.dcs_climate.build.wall_skarn":"矽卡岩墙",
+  "block.dcs_climate.build.wall_hornfels":"角页岩墙",
+  "block.dcs_climate.build.wall_marble":"大理石墙",
+  "block.dcs_climate.build.wall_schist_blue":"蓝片岩墙",
+  "block.dcs_climate.build.wall_schist_nether":"下界片岩墙",
+  "block.dcs_climate.build.wall_granite":"黑花岗岩墙",
+
+  "block.dcs_climate.build.stairs_beech_common":"山毛榉楼梯",
+  "block.dcs_climate.build.stairs_beech_walnut":"核桃楼梯",
+  "block.dcs_climate.build.stairs_beech_sweet":"白橡楼梯",
+  "block.dcs_climate.build.stairs_cherry_wild":"樱桃楼梯",
+  "block.dcs_climate.build.stairs_cinnamon_camphor":"香樟楼梯",
+  "block.dcs_climate.build.stairs_citrus_pomelo":"柑橘楼梯",
+  "block.dcs_climate.build.stairs_morus_mulberry":"桑木楼梯",
+  "block.dcs_climate.build.stairs_olive_ash":"白蜡楼梯",
+  "block.dcs_climate.build.stairs_palm_coconut":"棕榈楼梯",
+  "block.dcs_climate.build.stairs_reed_sorghum":"高粱茅草楼梯",
+
+  "block.dcs_climate.build.slab_beech_common":"山毛榉台阶",
+  "block.dcs_climate.build.slab_beech_walnut":"核桃台阶",
+  "block.dcs_climate.build.slab_beech_sweet":"白橡台阶",
+  "block.dcs_climate.build.slab_cherry_wild":"樱桃台阶",
+  "block.dcs_climate.build.slab_cinnamon_camphor":"香樟台阶",
+  "block.dcs_climate.build.slab_citrus_pomelo":"柑橘台阶",
+  "block.dcs_climate.build.slab_morus_mulberry":"桑木台阶",
+  "block.dcs_climate.build.slab_olive_ash":"白蜡台阶",
+  "block.dcs_climate.build.slab_palm_coconut":"棕榈台阶",
+  "block.dcs_climate.build.slab_reed_sorghum":"高粱茅草台阶",
+
+  "block.dcs_climate.build.fence_beech_common":"山毛榉栅栏",
+  "block.dcs_climate.build.fence_beech_walnut":"核桃栅栏",
+  "block.dcs_climate.build.fence_beech_sweet":"白橡栅栏",
+  "block.dcs_climate.build.fence_cherry_wild":"樱桃栅栏",
+  "block.dcs_climate.build.fence_cinnamon_camphor":"香樟栅栏",
+  "block.dcs_climate.build.fence_citrus_pomelo":"柑橘栅栏",
+  "block.dcs_climate.build.fence_morus_mulberry":"桑木栅栏",
+  "block.dcs_climate.build.fence_olive_ash":"白蜡栅栏",
+  "block.dcs_climate.build.fence_palm_coconut":"棕榈栅栏",
+  "block.dcs_climate.build.fence_reed_sorghum":"高粱茅草栅栏",
+
+  "block.dcs_climate.build.flintbricks":"燧石砖",
+  "block.dcs_climate.build.flintbricks_black":"黑色燧石砖",
+  "block.dcs_climate.build.flintbricks_white":"白色燧石砖",
+  "block.dcs_climate.build.flintbricks_red":"红色燧石砖",
+
+  "block.dcs_climate.build.mosaic_black":"黑色马赛克砖",
+  "block.dcs_climate.build.mosaic_blue":"蓝色马赛克砖",
+  "block.dcs_climate.build.mosaic_yellow":"黄色马赛克砖",
+  "block.dcs_climate.build.mosaic_red":"红色马赛克砖",
+
+    "## entities":"",
+
+  "entity.dcs_climate.bread_round":"圆面包",
+  "entity.dcs_climate.bread_square":"吐司面包",
+  "entity.dcs_climate.bread_cream":"奶油夹心面包",
+  "entity.dcs_climate.bread_sausage":"香肠面包",
+  "entity.dcs_climate.sandwich":"三明治",
+  "entity.dcs_climate.stick_beef":"牛肉串",

```

```suggestion
  "entity.dcs_climate.stick_beef":"烤肉串",
```
①两种肉串的原文均为`串焼き`，不存在差别
②上文是`烤牛肉串` `烤猪肉串`等，如果需要做区分请和上文一致

除此之外这里键顺序也和日文不一致，建议实际去游戏里跑一跑（因为下文存在键名不一致的问题，顺手提出来了